docs/examples/gluster-s3-storage-template/README.md

```diff
@@ -200,3 +203,21 @@ s3curl.pl --debug --id "testvolume:adminuser" --key "itsmine"  -- -k -v -s http:
 s3curl.pl --debug --id "testvolume:adminuser" --key "itsmine" --delete  --  http://$s3_storage_url/bucket1
 ```
 
+### To add a new user to the S3 account:
+
+#### First login to the pod
+```
+oc rsh <glusters3pod>
+```
+
+#### This step prepares the gluster volume where gswauth will save its metadata
+```
+gswauth-prep -A http://<ipaddr>:8080/auth -K gswauthkey
+
+Where, ip address of the glusters3 pod obtained from 'oc get pods -o wide'
+```
```

Comment:
<<  Is there a reason you reverted your variable name change commit?
This needs changes in Dockerfile and few scripts in side docker container.